package/MDAnalysis/core/universe.py

```diff
@@ -316,7 +321,8 @@ class Universe(object):
     """
     def __init__(self, topology=None, *coordinates, all_coordinates=False,
                  format=None, topology_format=None, transformations=None,
-                 guess_bonds=False, vdwradii=None, in_memory=False,
+                 guess_bonds=False, vdwradii=None,fudge_factor=0.55,
+                 lower_bound=0.1, in_memory=False,
```

Comment:
```suggestion
                 guess_bonds=False, vdwradii=None, fudge_factor=0.55,
                 lower_bound=0.1, in_memory=False,
```
Missing a space here

Reply:
How did one space make another two tests fail :smiling_face_with_tear: